pom.xml

```diff
@@ -2431,6 +2432,7 @@
                 <!-- Auto-generated target directories -->
                 <exclude>target/**</exclude>
                 <exclude>pinot-*/target/**</exclude>
+                <exclude>build/**</exclude> <!-- symbolic link pointing to pinot/pinot-distribution/target/apache-pinot-1.3.0-SNAPSHOT-bin/apache-pinot-1.3.0-SNAPSHOT-bin -->
```

Comment:
Note for reviewers: This symlink is not followed in Windows, so the license plugin consider it an actual file without valid license 😆

Reply:
let me remove this one, Windows is a separate topic ;)